src/components/standard/article.tsx

```diff
@@ -82,8 +83,15 @@ interface Props {
 const Standard: FC<Props> = ({ item, children }) => {
     // client side code won't render an Epic if there's an element with this id
     const epicContainer = item.shouldHideReaderRevenue
-        ? <div id="epic-container"></div>
-        : null
+        ? null
+        : <div id="epic-placeholder">
+            <Epic
```

Comment:
Ah, I think this may be causing us some errors on the server. The epic code imports and uses the Bridget comms layer, which doesn't seem to work server-side due to reliance on `window` amongst other things...

Reply:
Did you see this in the logs? it seems to have deployed ok. I was seeing these problems but adding a `try catch` around the window had resolved it locally.

Ideally we wouldn't want to render an Epic component server side, but I added it so that the styles would be added to the CSP.

Reply:
It's spitting out several errors when running locally for me.

Does the native comms code run anything as a side effect on import? If not we could pass a flag to the `Epic` component to tell it to avoid calling that code when running on the server.